selenium/src/main/java/org/exoplatform/platform/qa/ui/selenium/locator/gatein/GateinLocator.java

```diff
@@ -1456,8 +1458,8 @@
 
   public static final String          ELEMENT_MSG_CONFIRM_DELETE1                             = "Are you sure you want to delete";
 
-  public static final String          ELEMENT_MSG_UPDATE_USER_PROFILE                         =
-                                                                      "The user profile has been updated.";
+  public static final SelenideElement    ELEMENT_MSG_UPDATE_USER_PROFILE                         =
+                                                                      $(byText("OK"));
```

Comment:
1-Don't modify an existed locator, it may impact other tests. if you can't reuse it, add a new one.
2- Wrong declaration of locator.